packages/driver/cypress/support/utils.js

```diff
@@ -72,6 +72,20 @@ export const assertLogLength = (logs, expectedLength) => {
   expect(logs.length).to.eq(expectedLength, `received ${logs.length} logs when we expected ${expectedLength}: [${receivedLogs}]`)
 }
 
+export const findCrossOriginLogs = (consolePropCommand, logMap, matchingOrigin) => {
+  const matchedLogs = Array.from(logMap.values()).filter((log) => {
```

Comment:
Curious, why do you need Array.from? Does this work?
```suggestion
  const matchedLogs = logMap.values().filter((log) => {
```

Reply:
In this case it doesn't since `values()` returns an iterator. We could use a `for of` loop or something similar to iterate over the values, but we lose the benefit of the `map`/`filter` functionality I think.

Reply:
ahhh, i think i was thinking this.
```suggestion
  const matchedLogs = Object.values(logMap).filter((log) => {
```

Reply:
If logMap were a plain object I think that'd work, but since it's an instance of a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) I don't think it does

Reply:
Main reason is always have the latest log without having to sift through an array that has multiple logs for the same log ID. I believe we can also accomplish this in an Object too if we prefer that.